StringCompression.py

```diff
@@ -0,0 +1,24 @@
+def stringCompression(s):
+    if not s:
+        return s
+    comp = []
```

Comment:
Is there a reason you made this a list instead of a string? You can append to strings using `+` the same as lists. The only reason to really use a list here would be if you were doing it in one line using list comprehension.